flutter/shell/platform/tizen/tizen_window_ecore_wl2.cc

```diff
@@ -469,6 +479,73 @@ void TizenWindowEcoreWl2::Show() {
   ecore_wl2_window_show(ecore_wl2_window_);
 }
 
+void TizenWindowEcoreWl2::UpdateFlutterCursor(const std::string& cursor_name) {
+#ifdef TV_PROFILE
+  int pointer_size = -1;
+  if (vconf_get_int(kSysMouseCursorPointerSizeVConfKey, &pointer_size) < 0) {
+    FT_LOG(Error) << "Failed to load cursor size.";
+  }
+
+  std::string input_cursor_name;
+  if (cursor_name == "basic") {
+    switch (pointer_size) {
+      case 0:  // Large.
+        input_cursor_name = "large_normal";
+        break;
```

Comment:
nit: How about not using the switch statement? You will have less number of lines with simple if-else and the code might be more readable.

Reply:
done